Extensions/ParticleSystem/ExtensionSubDeclaration3.cpp

```diff
@@ -350,17 +357,4 @@ void ExtensionSubDeclaration3(gd::ObjectMetadata& obj) {
                     _("Setup"),
                     "CppPlatform/Extensions/particleSystemicon16.png")
       .AddParameter("object", _("Object"), "ParticleEmitter", false);
-  obj.AddExpression("ParticleAngle1",
```

Comment:
if we remove them, is this gonna break games which are using them?

Reply:
There is no function set for them so they were never available